docs/take-to-prod.md

```diff
@@ -0,0 +1,108 @@
+Taking your Spark .Net Application to Production
+===
+
+# Table of Contents
+This how-to provides general instructions on how to take your .NET for Apache Spark application to production.
+In this documentation, we will summary the most commonly asked scenarios when running Spark .Net Application.
+And you will also learn how to package your application and submit your application with [spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) and [Apachy Livy](https://livy.incubator.apache.org/).
+- [How to take your application to production when you have single dependency](#how-to-take-your-application-to-production-when-you-have-single-dependency)
```

Comment:
```suggestion
- [How to take your application to production when you have a single dependency](#how-to-take-your-application-to-production-when-you-have-a-single-dependency)
```

Not sure if we can change the phrasing here and still have it be precise, but "a single dependency" might sound a little cleaner.

Reply:
Alternatively, could we make these headings either more concise or more precise? i.e., either remove the "How to take your application to production" part since that phrase is already in the article title, or add a phrase that more specifically states what it means to take an app to production (does it just mean running `spark-submit`, so we could say something like "Deploy app with a single dependency"?).

```suggestion
- [Single dependency](#single-dependency)
```

```suggestion
- [How to deploy your application when you have a single dependency](#how-to-deploy-your-application-when-you-have-a-single-dependency)
```

Reply:
Thanks for your suggestion! I would prefer the second one which I think is concise and precise.